src/main/java/com/sap/sgs/phosphor/fosstars/model/feature/oss/OssFeatures.java

```diff
@@ -216,4 +219,19 @@ private OssFeatures() {
    */
   public static final Feature<Boolean> SIGNS_ARTIFACTS
       = new BooleanFeature("If a project signs artifacts");
+
+  /**
+   * If OWASP Dependency plugin is used to scan the project. It is either used as a mandatory step,
+   * optional step or not used at all.
+   */
+  public static final Feature<Status> OWASP_DEPENDENCY_CHECK_SCAN_AVAILABILITY =
+      new AvailabilityFeature("OWASP Dependency Check scan mode");
+
+  /**
+   * Check if configuration to fail the build exists in OWASP Dependency. Identify the CVSS
+   * threshold score to fail the build on finding vulnerabilities greater than the threshold.
+   */
+  public static final Feature<Double> OWASP_DEPENDENCY_CHECK_FAIL_CVSS_THRESHOLD =
+      new DoubleFeature(
```

Comment:
Would you mind introducing a new `BoundedDoubleFeature`? Like the following:

https://github.com/SAP/fosstars-rating-core/blob/master/src/main/java/com/sap/sgs/phosphor/fosstars/model/feature/BoundedIntegerFeature.java